.github/workflows/build.yml

```diff
@@ -89,10 +97,22 @@ jobs:
         if: matrix.os == 'windows'
         run: |
           codecov.exe -f ./out/tests/coverage-unit.txt -v --flag os_windows
-      - name: Build
-        run: make build
+      - name: Build Unix
+        if: runner.os != 'Windows'
+        run: |
+          echo "*Building*"
+          echo "${PACK_VERSION}"
+          make build
         env:
           PACK_BUILD: ${{ github.run_number }}
+        shell: bash
+      - name: Build Windows
+        if: runner.os == 'Windows'
+        run: |
+          make build
```

Comment:
Why not print the version here too?

Reply:
Ya just going to remove this,  running the action will show these in the log already.